### ISSUE TITLE
RobertaQA_not_Torchscript model gives blank prediction output

### ISSUE THREAD

**Issue by shivamshriwas** [pytorch/serve#466]
RobertaQA_Torchscript generates output for the sample text file whereas RobertaQA_not_Torchscript returns an empty string prediction. Refer to below logs.

Input file: [examples/Huggingface_Transformers/QA_artifacts/sample_text.txt](https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/QA_artifacts/sample_text.txt) 

```
ubuntu@ip-172-31-63-5:~/serve$ curl -X GET "http://localhost:8081/models"
{
  "models": [
    {
      "modelName": "RobertaQA_Torchscript",
      "modelUrl": "https://torchserve.s3.amazonaws.com/mar_files/roberta_qa_torchscript.mar"
    },
    {
      "modelName": "RobertaQA_not_Torchscript",
      "modelUrl": "https://torchserve.s3.amazonaws.com/mar_files/roberta_qa_no_torchscript.mar"
    }
  ]
}
ubuntu@ip-172-31-63-5:~/serve$ curl -X POST http://127.0.0.1:8080/predictions/RobertaQA_not_Torchscript -T examples/Huggingface_Transformers/QA_artifacts/sample_text.txt 
ubuntu@ip-172-31-63-5:~/serve$ 
ubuntu@ip-172-31-63-5:~/serve$ 
ubuntu@ip-172-31-63-5:~/serve$ curl -X POST http://127.0.0.1:8080/predictions/RobertaQA_Torchscript -T examples/Huggingface_Transformers/QA_artifacts/sample_text.txt 
</s></s> the weather is nice,ubuntu@ip-172-31-63-5:~/serve$ 
```

As per the input file given to the above models for [regression test case](https://github.com/pytorch/serve/blob/master/test/postman/inference_data.json) following behavior was observed.
input file : [examples/Huggingface_Transformers/sample_text.txt](https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/sample_text.txt)
```
ubuntu@ip-172-31-63-5:~/serve$ curl -X POST http://127.0.0.1:8080/predictions/RobertaQA_Torchscript -T examples/Huggingface_Transformers/sample_text.txt 
 jubuntu@ip-172-31-63-5:~/serve$ 
ubuntu@ip-172-31-63-5:~/serve$ 
ubuntu@ip-172-31-63-5:~/serve$ curl -X POST http://127.0.0.1:8080/predictions/RobertaQA_not_Torchscript -T examples/Huggingface_Transformers/sample_text.txt 
ubuntu@ip-172-31-63-5:~/serve$ 
```
Also, note RobertaQA_Torchscript returned letter **" j "** .  Is this expected?

Expected behavior: Scripted and eager mode model should have similar output.
If the input files used are not correct, then test case and documentation should be updated

**Comment by HamidShojanazeri** [comment]
It has been fixed in PR#474.

**Comment by HamidShojanazeri** [comment]
PR #996 will address this issue.
